components/n-ui/tracking/third-party/floodlight.js

```diff
@@ -2,19 +2,15 @@
 module.exports = function (flags) {
 
 	const isAnonymous = !/FTSession/.test(document.cookie);
-	const spoorId = /spoor-id=([^;]+)/.exec(document.cookie);
-	const host = 'https://4235225.fls.doubleclick.net/activityi;src=4235225';
+	const spoor = /spoor-id=([^;]+)/.exec(document.cookie);
+	const spoorId = (spoor && spoor.length > 0) ? spoor[1] : '';
 
 	// sign-up attributes
 	const c = document.querySelector('[data-signup-country]');
```

Comment:
Where are these being used?

Reply:
Oops... how did those get back in there. Good spot 👍